pom.xml

```diff
@@ -93,6 +94,11 @@
                 <artifactId>okta-sdk-okhttp</artifactId>
                 <version>1.6.0-SNAPSHOT</version>
             </dependency>
+            <dependency>
+                <groupId>com.squareup.okhttp3</groupId>
+                <artifactId>okhttp</artifactId>
+                <version>4.4.1</version>
```

Comment:
why was okhttp added?

Reply:
okhttp client is used as it provides a very elegant way of making rest requests. I preferred this as this is more easier and direct.

Reply:
Will remove this dep.